src/pages/portfolio/AccountOverviewSection.tsx

```diff
@@ -0,0 +1,182 @@
+import { useCallback } from 'react';
+
+import { shallowEqual } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { ButtonAction } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+import { AppRoute } from '@/constants/routes';
+import { ColorToken } from '@/constants/styles/base';
+
+import { useBreakpoints } from '@/hooks/useBreakpoints';
+import { useStringGetter } from '@/hooks/useStringGetter';
+import { useLoadedVaultAccount, useLoadedVaultDetails } from '@/hooks/vaultsHooks';
+
+import { Button } from '@/components/Button';
+import { Output, OutputType } from '@/components/Output';
+import { NewTag, Tag, TagSign, TagType } from '@/components/Tag';
+import { WithLabel } from '@/components/WithLabel';
+
+import { getSubaccount } from '@/state/accountSelectors';
+import { useAppSelector } from '@/state/appTypes';
+
+import { runIf } from '@/lib/do';
+import { isTruthy } from '@/lib/isTruthy';
+import { testFlags } from '@/lib/testFlags';
+import { orEmptyObj } from '@/lib/typeUtils';
+
+export const MegavaultYieldTag = () => {
+  const stringGetter = useStringGetter();
+  const vault = useLoadedVaultDetails().data;
+  if (vault?.thirtyDayReturnPercent == null || vault.thirtyDayReturnPercent < 0.02) {
+    return <NewTag>{stringGetter({ key: STRING_KEYS.NEW })}</NewTag>;
+  }
+
+  return (
+    <Tag type={TagType.Number} sign={TagSign.Positive}>
+      <Output type={OutputType.Percent} value={vault?.thirtyDayReturnPercent} />
+    </Tag>
+  );
+};
+
+export const AccountOverviewSection = () => {
+  const navigate = useNavigate();
+  const stringGetter = useStringGetter();
+
+  const { equity, freeCollateral } = orEmptyObj(useAppSelector(getSubaccount, shallowEqual));
+  const { balanceUsdc: vaultBalance } = orEmptyObj(useLoadedVaultAccount().data);
+  const totalValue = runIf(equity?.current, (e) => e + (vaultBalance ?? 0));
+
+  const handleViewVault = useCallback(() => {
+    navigate(`${AppRoute.Vault}`, {
+      state: { from: AppRoute.Portfolio },
+    });
+  }, [navigate]);
+
+  const showVaults = testFlags.enableVaults;
+  const { isTablet } = useBreakpoints();
+
+  const pieSections = [
+    {
+      id: 'free-collateral',
+      label: stringGetter({ key: STRING_KEYS.FREE_COLLATERAL }),
+      amount: freeCollateral?.current,
+      color: ColorToken.GrayPurple2,
+    },
+    {
+      id: 'open-positions',
+      label: stringGetter({ key: STRING_KEYS.OPEN_POSITIONS }),
+      amount: runIf(equity?.current, freeCollateral?.current, (e, f) => e - f),
+      color: ColorToken.Green2,
+    },
+    (showVaults || (vaultBalance ?? 0) > 0.01) && {
```

Comment:
uhhh I feel like vaults is showing all the time on the staging link 👀 which seems unintentional - this line is probably suspect?

Reply:
the vault testFlag is set to true on staging. I think this was an ask from PMs